contrib/node/src/python/pants/contrib/node/tasks/node_test.py

```diff
@@ -65,14 +70,29 @@ def _execute(self, all_targets):
     for target in targets:
       node_path = node_paths.node_path(target.dependencies[0])
 
-      args = ['run-script', target.script_name, '--'] + self.get_passthru_args()
-
-      with pushd(node_path):
-        self._currently_executing_test_targets = [target]
-        result, npm_test_command = self.execute_npm(args, workunit_labels=[WorkUnitLabel.TEST])
-        if result != 0:
-          raise TaskError('npm test script failed:\n'
-                          '\t{} failed with exit code {}'.format(npm_test_command, result))
+      logger.debug('First dependency: %s', target.dependencies[0])
```

Comment:
1) Should be a separate issue / PR.  Maybe we should enforce there is only one dependency for node_test
2) Yes, indeed you will need to have a separate node_test target because the script-name is passed in and can be different for each node_module.